plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/worker/KtLintWorkAction.kt

```diff
@@ -27,6 +31,16 @@ abstract class KtLintWorkAction : WorkAction<KtLintWorkAction.KtLintWorkParamete
 
     private val logger = Logging.getLogger("ktlint-worker")
 
+    private val androidProperty = UsesEditorConfigProperties.EditorConfigProperty(
+        type = PropertyType.LowerCasingPropertyType(
+            "android",
+            "A boolean value indicating that the project is an Android one.",
+            PropertyType.PropertyValueParser.IDENTITY_VALUE_PARSER,
+            emptySet()
+        ),
+        defaultValue = "false"
+    )
```

Comment:
Can't this be inferred?

Reply:
It seemed to me you were treating this property as a `String` but it seems feasible to be "inferred" when looking at how the `indentSize` property is handled in the [ktlint API](https://github.com/pinterest/ktlint/blob/master/ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/api/UsesEditorConfigProperties.kt#L197).

I can give it a try.